.travis.yml

```diff
@@ -5,7 +5,6 @@ python:
   - 3.4
   - 3.5
   - 3.6
-  - 3.7
```

Comment:
Instead of removing this, we can put it under the "allow_failures" section until I figure out why it's causing the build to fail

Reply:
Fixed this in my version. I'll push this to my master sometime soon.